docs/configuration.md

```diff
@@ -30,3 +32,19 @@ The following environment variables are available:
 | `KUBE_API_QPS` | QPS to use for the Kubernetes API client. See [Config.QPS](https://pkg.go.dev/k8s.io/client-go/rest#Config.QPS). A value of 0 or lower will use the default from client-go, which currently is 5. Default is 0. |
 | `TERMINATION_LOG_PATH` | Path of the termination log. This is where controller application will write the reason of its termination. Default value is `/dev/termination-log`. |
 | `GIT_ENABLE_REWRITE_RULE` | Enable Git wrapper to setup a URL `insteadOf` Git config rewrite rule for the respective source URL hostname. Default is `false`. |
+
+## Role-based Access Control
+
+The release deployment YAML file includes two cluster-wide roles for using Shipwright Build objects.
+The following roles are installed:
+
+- `shpwright-build-aggregate-view`: this role grants read access (get, list, watch) to most Shipwright Build objects.
+  This includes `BuildStrategy`, `ClusterBuildStrategy`, `Build`, and `BuildRun` objects.
+  This role is aggregated to the Kubernetes "view" role.
```

Comment:
```suggestion
  This role is [aggregated to the Kubernetes "view" role](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#aggregated-clusterroles).
```

Reply:
Thanks for the suggestion - I think the "Default user roles" section of that article may be a better fit:

https://kubernetes.io/docs/reference/access-authn-authz/rbac/#default-roles-and-role-bindings